stronk/models/workout_exercise.py

```diff
@@ -121,7 +121,7 @@ def update(self, attrs):
 
         newWorkoutsWeights = attrs.get('workout_weights')
         newWorkoutReps = attrs.get('workout_reps')
-       
+
```

Comment:
Not a big deal but if the only change here is a whitespace change couldn't we just make this the next time we make updates to the model?

Reply:
Good catch! I don't think there should even be a whitespace change. Lemme change that